keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -232,6 +260,29 @@ message ContainerUser {
 }
 ```
 
+#### features in StatusResponse which contains supplemental_groups_policy field
+
+To propagate whether the runtime supports fine-grained supplemental group control to `NodeFeatures.SupplementalGroupsPolicy`, it proposes to add a corresponding field`features` in `StatusResponse`. 
+
+```proto
+// service RuntimeService {
+// ...
+//     rpc Status(StatusRequest) returns (StatusResponse) {}
+// }
+message StatusResponse {
+...
+    // features describes the set of features implemented by the CRI implementation.
+    RuntimeFeatures features = ?;
```

Comment:
nit: should this be NodeFeatures as well?

Reply:
In CRI, I would like to use "Runtime" instead of "Node" because CRI does not use "Node" wording. WDYT??

Reply:
@haircommander fixed in [e054a20](https://github.com/kubernetes/enhancements/pull/4728/commits/e054a20cc6f36a14c500c08db6d79c7dc29a9732)